app/src/main/kotlin/compiler/intermediate/generators/SysV64CallingConvention.kt

```diff
@@ -27,16 +30,19 @@ object SysV64CallingConvention {
         // Add call instruction to actually call a given function
         cfgBuilder.addLinksFromAllFinalRoots(
             CFGLinkType.UNCONDITIONAL,
-            IFTNode.Call(targetFunction)
+            IFTNode.Call(targetFunction, usedRegisters, callerSavedRegisters)
```

Comment:
All these various words "used", "defined" and "saved" used interchangeably are somewhat confusing. Maybe just use "read" and "written"?

Reply:
Is it that confusing? I would say it's less confusing this way as these words have slightly different meanings. The terms "caller-saved" and "callee-saved" are part of calling convention docs (I consider them well-known) and they are more high-level terms (they describe function as a whole). On the other side, the meaning of "used" and "defined" is taken from liveness problem, and we use them to describe single instructions.
Here, variable `usedRegisters` hold list of registers which are going to be **used** by Call instruction.

Reply:
Yeah, makes sense.